uaclient/api/api.py

```diff
@@ -19,6 +19,7 @@
     "u.pro.attach.magic.wait.v1",
     "u.pro.attach.token.full_token_attach.v1",
     "u.pro.detach.v1",
+    "u.pro.token_info.v1",
```

Comment:
Let's keep this in alphabetical order (with the exception of `version` at the top and all non `pro` things at the bottom)